driver/normalizer/pyast/pyast.go

```diff
@@ -1,143 +1,172 @@
 // Package pyast defines constants from Python 2 and 3 AST.
 package pyast
 
-// GENERATED BY python-driver/native/gogen/gogen.py
```

Comment:
No longer autogenerated?

Reply:
No, actually it was being manually managed since some time, I moved the `HasInternalType` calls these to have a clearer `annotations.go` file (like you did in the Python driver) and sorted alphabetically. Also, I was getting slightly different results depending on the minor Python version used. I'll update or remove `gogen.py` soon.